docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/02_CMS_Architecture.md

```diff
@@ -753,7 +753,29 @@ The URL endpoints `{$AdminURL}mytabs/tab1` and `{$AdminURL}mytabs/tab2`
 should return HTML fragments suitable for inserting into the content area,
 through the `PjaxResponseNegotiator` class (see above).
 
+### Lazy loading fields on tab activation
+            
+When a tab is not lazy loaded via ajax, it might still be necessary to 
+delay some work (eg: when doing http requests) until the tab is activated. This is how, for example, the
+[GridFieldLazyLoader](https://api.silverstripe.org/4/SilverStripe/Forms/GridField/GridFieldLazyLoader.html) works.
+            
+In order to open up the same kind of features to other fields, a custom event is fired on all nodes with the `lazy-loadable` class inside the activated tab panel.
+They will receive a `lazyload` event that can be listened to in the following way:
+            
+```js
+el.addEventListener(
```

Comment:
$(el).one('lazyload') should achieve the same results, but since this is expected to be used in third party modules, i don't see the issue not using jquery or entwine. Actually, that should be real goal: provide platform agnostic support so that it works with anything, no ?
I'm not even sure how that would work with entwine (i don't think it's possible to listen to a custom event only once ?)

Reply:
I've been thinking about this - I think the main issue is that if you just drop this pure js into `LeftAndMain.extra_requirements_javascript` you won't even be able to find the listener unless it's in an entwine `onmatch` (except for when you directly load the form the tab is in via URL). So we probably should add that context to the example. And by that point you're already in a jquery context anyway.